test/__snapshot__/ecma262/miscellaneous/should-pass/gen/stand-alone/x002fx002a_x002ax002f_x002fx002a_optional_SingleLineDelimitedCommentSequence_x002ax002f--x003ethe_comment_extends_to_these_chara.md

```diff
@@ -39,6 +39,9 @@
 
 ```javascript
 
+/*
+  */
+/* optional SingleLineDelimitedCommentSequence */
 ```
 
```

Comment:
@aladdin-add Look at this test. It's an 1:1 of original source. This is what will happen when we track whitespace in the correct way.